x/staking/types/authz.go

```diff
@@ -3,61 +3,64 @@ package types
 import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
-	authz "github.com/cosmos/cosmos-sdk/x/authz/exported"
+	"github.com/cosmos/cosmos-sdk/x/authz"
 )
 
 // TODO: Revisit this once we have propoer gas fee framework.
 // Tracking issues https://github.com/cosmos/cosmos-sdk/issues/9054, https://github.com/cosmos/cosmos-sdk/discussions/9072
 const gasCostPerIteration = uint64(10)
 
+// Normalized Msg type URLs
 var (
 	_ authz.Authorization = &StakeAuthorization{}
 )
 
+var _ authz.Authorization = &StakeAuthorization{}
+
```

Comment:
These changes seem weird

Reply:
why it seams weird? I'm not sure, probably `lint-fix` updated it.

Reply:
It's duplicated code, and the comment doesn't make sense.

Reply:
ah, I is see. will remove the line.